stdlib/Profile/src/Profile.jl

```diff
@@ -1239,6 +1239,25 @@ function warning_empty(;summary = false)
     end
 end
 
+
+"""
+    Profile.take_heap_snapshot(io::IOStream)
+    Profile.take_heap_snapshot(filepath::String)
```

Comment:
add docs for `all_one` here

but I don't know if there might be a more descriptive name? or perhaps just a post-processing script (it seems easy to replace all nodes size > 0 with 1, since they are all at fixed locations in the JSON array)?

Reply:
the link to the docs also need to be added to the `stdlib/Profile/docs/src/index.md` file